deploy/crds/bundle.yaml

```
@@ -6926,6 +6926,71 @@ spec:
+                            BaseURL is the TrueFoundry control-plane URL, e.g.
+                            https://your-cluster.tfy-usea1-ctl.example.com. The provider appends
+                            /v1/control-plane/secret to this when fetching a secret.
+                          minLength: 1
+                          type: string
+                      required:
+                      - auth
+                      - baseURL
+                      type: object
+                    vault:
+                      description: Vault configures this store to sync secrets using
                     vault:
```

YAMLlint (1.38.0)

[error] duplication of key "vault" in mapping

(key-duplicates)